understanding/intro.html

```diff
@@ -343,9 +293,9 @@ <h3>Success Criteria</h3>
                   			
                   <section id="sufficient-techniques-advisory-techniques-and-failures">
                      				
-                     <h3>Sufficient Techniques, Advisory Techniques, and Failures</h3>
+                     <h3>十分な達成方法及び参考達成方法、及び失敗例</h3>
                      				
-                     <p>The next section, <em><a href="understanding-techniques">Understanding Techniques for WCAG Success Criteria</a></em>, provides important information about the techniques.
+                     <p>次の節 <em><a href="understanding-techniques">WCAG 達成基準の達成方法を理解する</a></em> では、達成方法に関する重要な情報を提供している。
```

Comment:
次節のタイトルですが、 https://github.com/waic/wcag22/pull/380 では「WCAG 達成基準のテクニックを理解する」としているので反映する必要があります。あと達成方法→テクニックとするなら:
```suggestion
                     <p>次の節 <em><a href="understanding-techniques">WCAG 達成基準のテクニックを理解する</a></em> では、テクニックに関する重要な情報を提供している。
```